person.rb

```diff
@@ -7,9 +8,10 @@ def initialize(id, age, name: 'unknown', parent_permission: true)
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
   end
 
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
- [ ] Since a person has many rentals and can create a rental, getters and setters for rentals should be present.